dictdiffer/utils.py

```diff
@@ -271,3 +275,24 @@ def are_different(first, second, tolerance):
         return abs(first-second) > tolerance * max(abs(first), abs(second))
     # we got different values
     return True
+
+
+def strip_removed_objects(obj):
+    """Strip all the values that are instance of ``RemovedObject``."""
+    if isinstance(obj, LIST_TYPES):
+        return [
+            strip_removed_objects(value)
+            for value in obj if not isinstance(value, RemovedObject)
+        ]
+    if isinstance(obj, DICT_TYPES):
+        new_dict = {}
+        for key, value in obj.items():
+            clean_value = strip_removed_objects(value)
```

Comment:
You might want to put this inside the if.

Reply:
That you don't need to `strip_removed_objects` of the `value` if it's a `RemovedObject`, so you can skip this line if `isinstance(value, RemovedObject)`, either moving it inside the following if, or inverting the if and continuing:
```python
for ...:
    if isinstance(value, RemovedObject):
        continue

    clean_value = ..
    new_dict[key] = clean_value
```